docs/install.rst

```diff
@@ -24,11 +24,21 @@ Portable                             See the `Windows portable version`_ section
                                         choco install streamlink
 
                                      `Installing Chocolatey packages`_
+                                     
+`Windows Package Manager`_           .. code-block:: bat
+
+                                       winget install streamlink
```

Comment:
Wrong indentation. The block content should be aligned with the first character of the block label, which means one space is missing.

Reply:
Missed this on my review, sorry @bastimeyer, shouldn't review PRs when it's almost time to sleep!